ci-operator/config/openshift/frr/openshift-frr-master__okd-scos.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.15_base-stream9:
```

Comment:
Can these references be to 4.16? We are doing all of the CentOS migration work in 4.16, so 4.15 might still contain RHEL bits.